### ISSUE TITLE
miniconda re-installation - specifications found to be in conflict

### ISSUE THREAD

**Issue by borealis88** [conda/conda#2567]
Hello, I am having trouble getting conda to work. Would appreciate any help. Am using Window 10.

I installed Miniconda3 and want to install libpython. When entering "conda install libpython" I get the following error message: "The following specifications were found to be in conflict", followed by a lengthy list, starting like this

  - backports (target=backports-1.0-py35_0.tar.bz2) -> python 2.7*|3.4*|3.5*
  - conda -> conda-env
  - conda -> menuinst
  - conda -> pycosat
  - conda -> python 2.7*|3.3*|3.4*
  - conda -> pyyaml
  - conda -> requests
  - conda-env (target=conda-env-2.4.5-py35_0.tar.bz2) -> python 2.7*|3.3*|3.4*|3.5*
  - decorator (target=decorator-4.0.9-py35_0.tar.bz2) -> python 2.6*|2.7*|3.3*|3.4*|3.5*
  - entrypoints (target=entrypoints-0.2-py35_1.tar.bz2) -> python 2.7*|3.4*|3.5*
  - get_terminal_size (target=get_terminal_size-1.0.0-py35_0.tar.bz2) -> backports
  - get_terminal_size (target=get_terminal_size-1.0.0-py35_0.tar.bz2) -> python 2.7*|3.4*|3.5*
  - ipykernel (target=ipykernel-4.3.1-py35_0.tar.bz2) -> ipython *|>=4.0
  - ipykernel (target=ipykernel-4.3.1-py35_0.tar.bz2) -> jupyter_client
  - ipykernel (target=ipykernel-4.3.1-py35_0.tar.bz2) -> python 2.7*|3.4*|3.5*
  - ipykernel (target=ipykernel-4.3.1-py35_0.tar.bz2) -> tornado >=4.0

(I have not included the entire list here)

I should also mention that I earlier had Anaconda2 and Anaconda3 installed but removed them as they also didn't work properly. For instance the Jupyter kernel didn't work.

I have not found a solution yet. Any help is greatly appreciated!

**Comment by msarahan** [comment]
There is no compatible libpython for Python 3.5. That is what your error
message is from.

The kernel issues are separate. Please create a new issue with more
information: versions of anaconda or Miniconda, error messages you see, and
anything else you think might be related. One common issue is having
another Python installed on your system,  and things getting mixed up.

**Comment by borealis88** [comment]
Thank you, msarahan for your response several weeks ago. I got held up with other things, so am only returning to this now. Appreciating any further help.

Here is the full error message received when entering "conda install libpython".

[Install_libpython_error_160716.txt](https://github.com/conda/conda/files/367080/Install_libpython_error_160716.txt)

What I had done earlier:
1. Installed Anaconda with Python 2.7
2. Decided to upgrade to 3.5, so uninstalled Anaconda/2.7 components.
3. Installed Anaconda with Python 3.5.
4. Was unable to install libpython
5. Uninstalled Anaconda, installed Miniconda 3
6. Unable to install libpython

Why do I want install libpython? I am trying to set up with Theano as described here. http://lepisma.github.io/articles/2015/07/30/up-with-theano-and-cuda/. The instruction is to install libpython.

In other words, I am unable to follow these instructions as I cannot install libpython. But the overarching issue is that I would like to set up Theano. In fact I was able to get it to work while I was still using python 2.7 - the problem started when I wanted to use 3.5 instead.

I am a novice to this, so greatly appreciating any help.
